github/lib.w

```diff
@@ -0,0 +1,7 @@
+bring "./probot/probot.w" as probot;
+
+pub class ProbotApp extends probot.ProbotApp {
```

Comment:
I am wondering if this needs library needs to be called `probot` instead of GitHub.

Reply:
Okay.

I guess the readme will be useful so I can understand how this is supposed to be used.